js/index.js

```diff
@@ -1,41 +1,86 @@
 import { Game } from './game.js';
 import { Player } from './player.js';
-import {get, getAll} from './util/utilities.js'
+import {get, getAll, createEl} from './util/utilities.js'
 
-
-//Notionally from DOM
-let playerOneName = 'Travis';
-let playerTwoName = 'TRobot';
-let playerThreeName = 'TJustins';
-const boardFromDOM = document.querySelectorAll('.game-board');
 let game;
+const boardFromDOM = document.querySelectorAll('.game-board');
+const domMethods = {
+  render
+}
 
-//create users array
-get('#start-btn').addEventListener('click', () => {
-  // playerOneName = get('#p1-name-input').value;
-  // playerTwoName = get('#p2-name-input').value;
-  // playerThreeName = get('#p3-name-input').value;
-
-  //set screen transition
-    //add transition img
-  get('#view').innerHTML = '<img src="./img/loading.gif">';
+get('#start-btn').addEventListener('click',transitionToGame );
+get('#view').addEventListener('click', render);
 
-  window.setTimeout(() => {
-    game = new Game([playerOneName, playerTwoName, playerThreeName]);
-  }, 500) 
-});
 
-get('#view').addEventListener('click', () => {
+function render(event) {
+  // console.log(event.target.id);
+  // switch (event.target.id) {
+  //   case 'start-btn': console.log('yup');
+  // }
```

Comment:
Meep meep, don't push up commented out code and console logs. They clutter up the diff.